packages/bot-updategas/src/GasUpdater.ts

```diff
@@ -71,6 +71,7 @@ export class GasUpdater {
     mangrove: Mangrove,
     acceptableGasGapToOracle: number,
     oracleSourceConfiguration: OracleSourceConfiguration,
+    private overEstimateOracleGasPriceByXPercent: number,
```

Comment:
Consistency would be good, so I think either this should follow the existing pattern or the rest should also be changed into this pattern.